frontend/src/components/MapView/Layers/BoundaryLayer/index.tsx

```diff
@@ -28,6 +28,65 @@ interface ComponentProps {
   before?: string;
 }
 
+export const getLayers = (layer: BoundaryLayerProps) => [
+  getLayerMapId(layer.id, 'fill'),
+];
+
+export const onClick = ({
+  dispatch,
+  layer,
+  t,
+}: MapEventWrapFunctionProps<BoundaryLayerProps>) => (
+  evt: MapLayerMouseEvent,
+) => {
+  const isPrimaryLayer = isPrimaryBoundaryLayer(layer);
+  if (!isPrimaryLayer) {
+    return;
+  }
+
+  const layerId = getLayerMapId(layer.id, 'fill');
+
+  // TODO: maplibre: fix feature
```

Comment:
@echaidemenos let's open an issue if you think it's important to get that sorted